### PR TITLE
Switch to yarn

### LINKED ISSUE #351: Switch from Bower to Yarn for JavaScript dependencies
We did this in `plots2` and it took a while to figure out, but ultimately wasn't too complex:

https://github.com/publiclab/plots2/pull/2659/files

Let's do this here too!

### PR DESCRIPTION
**Pull request by kaustubh-nair**
Fixes #351

### CODE REVIEW

**Comment by kaustubh-nair** [comment]
@publiclab/mapknitter-reviewers Please review this. Thanks!

File: start.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 pidfile=/app/tmp/pids/server.pid
 
 bundle check || bundle install
-bower install --allow-root
+yarn install --allow-root
```
Comment (alaxalves) [inline start.sh]: Is there such flag on yarn??

Comment (kaustubh-nair) [reply start.sh]: No there isn't. Removed it. Thanks :+1:

**Comment by kaustubh-nair** [comment]
Hi I've merged this to https://github.com/publiclab/mapknitter/pull/669 and also published `development` to `unstable`

**Pull request merged**